config/localization/bundles/src/bundle/org/sakaiproject/localization/bundle/tool/tools_eu.properties

```diff
@@ -210,6 +210,8 @@ sakai.commons.title=Commons
 sakai.commons.description=Sare sozial erako tresna bat da, thumbail-erako url-ak dituen bidalketak ahalbidetzen dituena. 
 sakai.meetings.description= A tool for online meetings management
 sakai.meetings.title= Meetings
+microsoft.mediagallery.description= A Microsoft tool to browse Media Files
```

Comment:
Hey @pablosanroman, this is a new tool, would you mind providing a translation for these texts? Thanks!

Reply:
Hi @mpellicer 

microsoft.mediagallery.description= Ikusentzunezko fitxategiak bilatzeko Microsoft-en tresna bat da hau. 
microsoft.mediagallery.title= Galeria

Reply:
Thanks!